src/clr-angular/button/button-group/_button-group.clarity.scss

```diff
@@ -15,6 +15,27 @@
     .btn {
       margin: 0;
       vertical-align: top;
+      overflow: visible;
+
+      label {
+        height: 100%;
+      }
+
+      input[type='radio']:focus,
+      input[type='checkbox']:focus {
+        appearance: none;
+        opacity: 1;
+        top: 0;
```

Comment:
I don't see any positioning absolute or relative. What's this `top: 0` doing?

Reply:
We use a mixin that sets the position to absolute with a non-zero top value:

https://github.com/vmware/clarity/blob/7562bbff2ddd251393cc652af9905766f3ca7f11/src/clr-angular/forms-deprecated/utils/_mixins.clarity.scss#L10-L23